src/SequenceDiagramViewerPage.tsx

```diff
@@ -57,7 +57,7 @@ export function SequenceDiagramViewerPage() {
         <SequenceDiagramViewer
           localUserId={debugLog.localUserId}
           selectedUserId={selectedUserId}
-          onSelectUserId={setSelectedUserId}
+          onSelectUserId={setSelectedUserId as (key: Key) => unknown}
```

Comment:
is there a better solution for this?
At some point we need it to be a `(key: Key)=>any` because that is required by `<SelectInput onSelectionChange={onSelectInputChangedFunctionOfTypeKeyKeyAny} />`

Reply:
Can't we use the `Dispatch<(prevState: undefined) => undefined>` type that's in GroupCallInspector.tsx?

Reply:
That is what I tried initially but it raises the follwoing error:
```
(property) SequenceDiagramViewerProps.onSelectUserId: (key: React.Key) => unknown
Type 'Dispatch<(prevState: undefined) => undefined>' is not assignable to type '(key: Key) => unknown'.
  Types of parameters 'value' and 'key' are incompatible.
    Type 'Key' is not assignable to type '(prevState: undefined) => undefined'.
      Type 'string' is not assignable to type '(prevState: undefined) => undefined'.ts(2322)
 ```
 So, at some point in time we need to convert it to  '(key: Key) => unknown` so that it is accepted by the `onOnSelectUserId`

Reply:
Ah, okay, in that case I would recommend changing the type of `onSelectUserId` to `(userId: string) => void` and do the typecast inside SequenceDiagramViewer instead, so that the component asserts in a single location that it always outputs strings

Reply:
That is a really nice solution. Thanks!